struct.go

```diff
@@ -74,22 +74,21 @@ type RegisterInstanceJson struct {
 	Instance *Instance `json:"instance"`
 }
 
-// Instance [de]serializeable [to|from] Eureka XML.
+// Instance [de]serializeable [to|from] Eureka [XML|JSON].
 type Instance struct {
-	XMLName          struct{} `xml:"instance" json:"-"`
-	HostName         string   `xml:"hostName" json:"hostName"`
-	App              string   `xml:"app" json:"app"`
-	IPAddr           string   `xml:"ipAddr" json:"ipAddr"`
-	VipAddress       string   `xml:"vipAddress" json:"vipAddress"`
-	SecureVipAddress string   `xml:"secureVipAddress" json:"secureVipAddress"`
+	HostName         string `xml:"hostName" json:"hostName"`
```

Comment:
When using an `xml.Encoder` to encode an element, one must supply an `xml.StartElement` that determines the element's name. The "XMLName" field has no effect in that case. I struggled with a bug here for a while, trying to figure out why my "XMLName" field's value wasn't being honored. I finally realized that the value passed to `xml.(*Encoder).EncodeElement` is what determines the element name.

If someone was setting this field's value to something other than "instance," the Eureka server would have rejected the resulting message body. If they were to try to set it now—if I restored the field to the `fargo.Instance` struct —it would have no effect. To leave it in would confuse both our maintainers and our users.

Now, it's possible that someone's code would stop compiling if they were accessing the "XMLName" field of `fargo.Instance`. I didn't think it was worth retaining this cruft just to preserve that illusion of compatibility. I think that this does warrant bumping at least the minor version number in the next release that would contain these changes.